charts/resoto/README.md

```diff
@@ -25,47 +25,51 @@ A Helm chart for Kubernetes
 | fullnameOverride | string | `""` |  |
 | image.pullPolicy | string | `"IfNotPresent"` |  |
 | image.repositoryFolder | string | `"somecr.io/someengineering"` |  |
-| image.tag | string | `""` |  |
+| image.tag | string | `""` | Overrides the image tag whose default is the chart appVersion. |
 | imagePullSecrets | list | `[]` |  |
 | nameOverride | string | `""` |  |
 | nodeSelector | object | `{}` |  |
 | podAnnotations | object | `{}` |  |
 | podSecurityContext | object | `{}` |  |
 | replicaCount | int | `1` |  |
-| resotocore.extraArgs | list | `[]` |  |
-| resotocore.extraEnv | list | `[]` |  |
-| resotocore.graphdb.database | string | `"resoto"` |  |
-| resotocore.graphdb.passwordSecret.key | string | `"password"` |  |
-| resotocore.graphdb.passwordSecret.name | string | `"arango-user"` |  |
-| resotocore.graphdb.server | string | `"http://single-server:8529"` |  |
-| resotocore.graphdb.username | string | `"resoto"` |  |
+| resotocore.extraArgs | list | `[]` | Use this section to define extra arguments |
+| resotocore.extraEnv | list | `[]` | Use this section to pass extra environment variables |
+| resotocore.graphdb | object | `{"database":"resoto","passwordSecret":{"key":"password","name":"arango-user"},"server":"http://single-server:8529","username":"resoto"}` | This defines the access to the graph database |
```

Comment:
does it make sense to define complex objects when all properties are described as well?

Reply:
This looks like the bug in the upstream doc generator, I checked this but couldn't find workaround. I am going to open an issue with them to follow up on this.

Reply:
I see. for me it's a bit confusing. let's see if this can be improved. fine for now.